apps/web/screens/Search/Search.tsx

```diff
@@ -292,14 +295,24 @@ const Search: Screen<CategoryProps> = ({
     return false
   }
 
+  const getItemLink = (item: SearchType) => {
+    if (
+      item.__typename === 'LifeEventPage' &&
+      item.pageType === 'Digital Iceland Service'
```

Comment:
wondering whether 'Life Event' and 'Digital Iceland Service' should be listed as constants?

Reply:
The GraphQL type for pageType is a string (instead of being 'Digital Iceland Service' | 'LifeEventPage'). But yeah having a constant might be the best middle ground :)